src/SpectatorView.Native/SpectatorView.Compositor/Compositor/VideoEncoder.cpp

```diff
@@ -392,8 +444,9 @@ void VideoEncoder::StopRecording()
     {
         while (!videoQueue.empty())
         {
-            videoQueue.pop();
+			videoQueue.pop();
         }
+		OutputDebugString(L"Cleared video queue\n");
```

Comment:
>OutputDebugString(L"Cleared video queue\n"); [](start = 2, length = 44)

Should this be in an #if _DEBUG block as well?

Reply:
OutputDebugString already only writes things to the console in debug mode, but i guess it still may result in string allocation, ill add it to a _DEBUG stub